packages/manager/apps/web-hosting/src/api/api.ts

```diff
@@ -0,0 +1,35 @@
+import { fetchIcebergV2, v6 } from '@ovh-ux/manager-core-api';
+
+export const getwebHostingAttachedDomainQueryKey = (shouldFetchAll?: boolean) =>
+  ['get', 'webhosting', 'attachedDomain', shouldFetchAll ? 'all' : ''].filter(
+    Boolean,
+  );
+export const getwebHostingAttachedDomain = async ({
+  pageParam,
+  pageSize = 15,
+}: {
+  pageParam?: string;
+  pageSize?: number;
+}) => {
+  const response = await fetchIcebergV2({
+    route: '/webhosting/attachedDomain',
+    pageSize,
+    cursor: pageParam,
+  });
+  return response;
```

Comment:
I think you can return directly the call of `fetchIcerbergV2` if you don't use `response` before returning it